cmake/environmentchecks.cmake

```diff
@@ -25,10 +25,15 @@ if (EMSCRIPTEN)
 else ()
   find_package(Python ${PREFERRED_PYTHON_VERSION} COMPONENTS Interpreter Development)
 
-  if(WIN32)
-    find_program(CLCACHE_EXE clcache)
-  else()
-    find_program(CCACHE_EXE ccache)
+  if(NOT WIN32 OR MSVC)
+    find_program(BUILDCACHE_EXE buildcache)
+  endif()
+  if(NOT BUILDCACHE_EXE)
+    if(MSVC)
+      find_program(CLCACHE_EXE clcache)
+    elseif(NOT WIN32)
```

Comment:
I don't see how this applies, I think you can have `ccache` on Windows with `mingw`.

Reply:
Agreed. Done in commit 3943f635.